### ISSUE TITLE
Ability to configure logging format

### ISSUE THREAD

**Issue by ewbankkit** [wascc/logging-provider#3]
It would be nice to have a way to configure the logging format.
For example the logging machinery of certain runtime environments already add their own timestamps to log records. AWS Lambda logs end up in CloudWatch and right now we get duplicate timestamps:

![Annotation 2020-03-27 160529](https://user-images.githubusercontent.com/2404182/77795978-dad99100-7044-11ea-89b6-93df6fed9e01.png)

**Comment by ewbankkit** [comment]
Another option would be to use the host's logger (which would have been configured to taste), but that would seem to require passing the host's [logger](https://docs.rs/log/0.4.10/log/fn.logger.html) to the capability provider at initialization time.

**Comment by ewbankkit** [comment]
This is resolved for my use case by linking this provider statically into a host (once https://github.com/wascc/logging-provider/pull/5 is merged) and thereby picking up the host's logger.
I'll close this one.
